docs/pages/database-access/reference/aws.mdx

```diff
@@ -5,10 +5,10 @@ description: AWS IAM policies for Teleport database access.
 
 ## Auto-discovery
 With the appropriate IAM permissions, Teleport automatically discovers and
-configures IAM policies for Amazon RDS and Redshift.
+configures IAM policies for Amazon RDS, Redshift and ElastiCache.
 
-Teleport also requires permission to update database configurations, for example, to
-enable IAM authentication on RDS databases.
+Teleport may also require permission to update database configurations, for
```

Comment:
Should we add a sentence here to help the reader understand when Teleport does/doesn't require permissions to update database configurations?